rhobs/alerting/data_plane/prometheus.seb_created_to_ready_alerts.yaml

```diff
@@ -1,35 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
```

Comment:
Anything related to SEB should be in separate commit/PR, don't mix unrelated fixes under the same commit.